### PR TITLE
Quick hack to get rid of  anything with "(historical)" in it

### PR DESCRIPTION
**Pull request by simonpoole**
Not pretty, but works.

This is a bit WIP, please wait with merging.

### CODE REVIEW

**Comment by simonpoole** [comment]
"OK" to merge now. I'm not really happy (the old proper noun vs name in a different language issue), but it should now be no worse than before.

File: topNames.json [hunk 1]
```diff
@@ -1,5256 +1,6160 @@
...
+    "shop/supermarket|U Express": 129,
+    "amenity/place_of_worship|St James": 110,
+    "shop/alcohol|Beer Store": 66,
+    "amenity/place_of_worship|St Michael and All Angels": 66,
+    "amenity/place_of_worship|Waldkapelle": 79,
+    "amenity/fast_food|Jack in the Box": 868,
+    "amenity/fast_food|KFC/Taco Bell": 70,
+    "amenity/pharmacy|Rosen Apotheke": 65,
+    "amenity/bank|Berliner Volksbank": 73,
+    "amenity/fuel|Max
```
Comment (bhousel) [inline topNames.json]: Looks like they are still in the index?

Comment (bhousel) [inline topNames.json]: sorry my mistake, topNames is the source data..

**Comment by bhousel** [comment]
Thanks @simonpoole!

**Pull request merged**